scripts/tools.js

```diff
@@ -0,0 +1,67 @@
+const path = require("path");
+const chalk = require("chalk");
+const shell = require("shelljs");
+const fs = require("fs");
+const elementsDir = path.join(__dirname, "../elements");
+
+module.exports.camelToKebab = string =>
+  string
+    .replace(/([a-z0-9])([A-Z])/g, "$1-$2")
+    .replace(/([A-Z])([A-Z])(?=[a-z])/g, "$1-$2")
+    .toLowerCase();
```

Comment:
You've got `lodash` as a project dependency so you could use [that](https://lodash.com/docs/4.17.15#kebabCase) instead of having to maintain a custom function. ⏳